domains/runtime/auto-id/src/lib.rs

```diff
@@ -480,7 +490,7 @@ construct_runtime!(
         BlockFees: pallet_block_fees = 91,
 
         // Sudo account
-        Sudo: pallet_sudo = 100,
+        Sudo: pallet_domain_sudo = 100,
```

Comment:
We should probably use another index for the `pallet_domain_sudo` instead of reusing `100` IMO

Reply:
I dont think it makes any difference. Any specific reason why we should use another ?

Reply:
not actually. Indexes are used by clients while constructing the extrinsics and the clients should not make an assumption but rather need to fetch the runtime Metadata to construct anything runtime related. So we are free to re-use the indexes here